input/core/src/commonMain/kotlin/majestic/ImageView.kt

```diff
@@ -0,0 +1,21 @@
+package majestic
```

Comment:
We do not need this Composable in Majestiv because it only abstract away just 1 parameter. So, please remove it